show/vlan.py

```diff
@@ -110,32 +110,29 @@ def config(db):
     keys = list(data.keys())
     member_data = db.cfgdb.get_table('VLAN_MEMBER')
 
+    def member_name(member):
+        alias = member
+        if clicommon.get_interface_naming_mode() == "alias" and member:
+            alias = clicommon.InterfaceAliasConverter(db).name_to_alias(member)
```

Comment:
Another improvement here would be to eliminate instantiating `InterfaceAliasConverter` for every member. Suggest instantiating it once at the top of the `config` function, like:

```python
iface_alias_converter = clicommon.InterfaceAliasConverter(db)
```

Also suggest changing the name of this nested function to `get_iface_name_for_display()`.

Reply:
You made an improvement to only call `clicommon.get_interface_naming_mode()` once at the top of the function, and that's good, but I was suggesting only calling `clicommon.InterfaceAliasConverter(db)` once and storing it in a local object. Suggest doing this as well.